gen/perunServicesInit.pm

```diff
@@ -143,7 +143,7 @@ sub getHashedHierarchicalData {
 	if(defined $local_data) { return $local_data; }
   my $filterExpiredMembers = shift;
   unless($filterExpiredMembers) { $filterExpiredMembers = 0; }
-  my $data = $servicesAgent->getHashedHierarchicalData(service => $service->getId, facility => $facility->getId, filterExpiredMembers => $filterExpiredMembers);
+  my $data = $servicesAgent->getHashedHierarchicalData(service => $service->getId, facility => $facility->getId, filterExpiredMembers => $filterExpiredMembers, consentEval => $ENV{"CONSENT_EVAL"});
```

Comment:
Again, couldn't we just create a global variable, include -c in the GetOptions call at the beginning of the init method to set it to true if needed and use it here?